app/models/rubric_criterion.rb

```diff
@@ -82,7 +81,9 @@ def set_default_levels
   #                      does not evaluate to a float, or if the criterion is not
   #                      successfully saved.
   def self.create_or_update_from_csv_row(row, assignment)
-    if row.length < RUBRIC_LEVELS + 2
+    # get number of required fields
+    required_fields = Level.validators.grep(ActiveRecord::Validations::PresenceValidator).length
```

Comment:
We can allow the user to upload a rubric criterion with no levels. So really the only thing to check here is that `working_row` has at least *one* element (the name of the criterion). So don't worry about the Level required fields here.